docs/features/migrate_to_pootle_fs.rst

```diff
@@ -34,10 +34,15 @@ following:
 .. image:: ../_static/pootle_fs_link.png
 
 
-* *Filesystem backend* to ``localfs``
-* *Backend URL or path* to the value of
+* **Filesystem backend** to ``localfs``
+* **Backend URL or path** to the value of
   :setting:`POOTLE_TRANSLATION_DIRECTORY` + MYPROJECT, e.g.
   :file:`/path/to/pootle/translations/MYPROJECT`
+* **Translation path mapping** to the one your project uses.
+
+  .. note:: You can instead pick one of the **Translation mapping presets** to
+     easily set your project's **Translation path mapping** in case your
+     project actually matches one of the available presets.
```

Comment:
I hate seeing repeated paragraphs, they're just open to get out of sync and out of date.  Do we need to explain something that is on the actual form? And I'm not sure why we need a note, that is to make something off topic or highlight, none of which is happening here. Just add "You can use one of the presets or specify your own mapping", but even that seems redundant as we're stepping them through a task list and that info is on the screen.

Reply:
I removed those in favor of simpler text.